test/evm/hardhat/chain-adapters/Arbitrum_Adapter.ts

```diff
@@ -49,9 +49,10 @@ let l1ERC20GatewayRouter: FakeContract,
   cctpMessenger: FakeContract,
   cctpTokenMinter: FakeContract,
   oftMessenger: FakeContract<IOFT>,
-  oftAddressBook: FakeContract<OFTAddressBook>;
+  adapterStore: FakeContract<AdapterStore>;
 
 const arbitrumChainId = 42161;
+const adapterId = arbitrumChainId;
```

Comment:
why distinguish between adapter ID's and arbitrum chain ID's at all? I think its more intuitive if you just use chain ID's everywhere and have no notion of an "adapter ID"